lib/rspec/sidekiq/matchers/have_enqueued_sidekiq_job.rb

```diff
@@ -1,11 +1,11 @@
 module RSpec
   module Sidekiq
     module Matchers
-      def have_enqueued_job(*expected_arguments)
-        HaveEnqueuedJob.new expected_arguments
+      def have_enqueued_sidekiq_job(*expected_arguments)
+        HaveEnqueuedSidekiqJob.new expected_arguments
       end
```

Comment:
The downsides remain the same as the current use (overriding the `rspec-rails` matcher) but the upside is support for using this gem as it was without `rspec-rails`, where as after merging this currently anyone using this gem without `rspec-rails` will encounter errors on a point change.